packages/core/src/extensions/SuggestionMenu/SuggestionPlugin.ts

```diff
@@ -127,7 +127,7 @@ class SuggestionMenuView<
       .focus()
       .deleteRange({
         from:
-          this.pluginState.queryStartPos! -
+          this.pluginState.queryStartPos!() -
```

Comment:
can the `!` here (and in usages below) cause errors if a remote user deletes a piece of text?